x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/total_feature_importance_summary/feature_importance_summary.tsx

```diff
@@ -188,9 +188,14 @@ export const FeatureImportanceSummaryPanel: FC<FeatureImportanceSummaryPanelProp
   const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = docLinks;
   const tickFormatter = useCallback((d) => Number(d.toPrecision(3)).toString(), []);
 
+  // do not expand by default if no feature importance data
+  const hasTotalFeatureImportance = useMemo(() => {
```

Comment:
Nit: this could be `() => totalFeatureImportance.length > 1`

Reply:
Updated here https://github.com/elastic/kibana/pull/79546/commits/0e1b212925bc907d5d9960dda20151826f432208